### PR TITLE
Add motivation 

### PR DESCRIPTION
**Pull request by awaelchli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
 
 # nanoGPT - with Fabric ⚡
 
-**This is Karpathy's nanoGPT repo with the addition of [Fabric](https://pytorch-lightning.readthedocs.io/en/latest/fabric/fabric.html).**
-**Fabric is fast, lightweight, and scales your models without boilerplate code.**
+**This is [Karpathy's nanoGPT](https://github.com/karpathy/nanoGPT) repo with the addition of [Fabric](https://pytorch-lightning.readthedocs.io/en/latest/fabric/fabric.html).**
+**Fabric the fast and lightweight way to scale your models without boilerplate code.**
+
+- Handles all the boilerplate device logic for you ()
```
Comment (lantiga) [inline README.md]: something missing in the brackets?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+- Easily switch from debugging on CPU to multi-GPU and multi-node training
```
Comment (lantiga) [inline README.md]: Can we mention mps as well?

**Review by lantiga — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,8 +1,15 @@
 
 # nanoGPT - with Fabric ⚡
 
-**This is Karpathy's nanoGPT repo with the addition of [Fabric](https://pytorch-lightning.readthedocs.io/en/latest/fabric/fabric.html).**
-**Fabric is fast, lightweight, and scales your models without boilerplate code.**
+**This is [Karpathy's nanoGPT](https://github.com/karpathy/nanoGPT) repo with the addition of [Fabric](https://pytorch-lightning.readthedocs.io/en/latest/fabric/fabric.html).**
```
Comment (Borda) [inline README.md:4]:
```suggestion
**This is [Karpathy's nanoGPT](https://github.com/karpathy/nanoGPT) repo with the addition of [Fabric](https://pytorch-lightning.readthedocs.io/en/1.9.0/fabric/fabric.html).**
```

**Review by Borda — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,8 +1,15 @@
...
+**Fabric the fast and lightweight way to scale your models without boilerplate code.**
```
Comment (justusschock) [inline README.md]:
```suggestion
**Fabric is the fast and lightweight way to scale your models without boilerplate code.**
```

File: README.md [hunk 5]
```diff
@@ -1,8 +1,15 @@
...
+
+- Handles all the boilerplate device logic for you
+- Easily switch from debugging on CPU to GPU (Apple M1/M2, NVIDIA CUDA etc.) or multi-GPU or even multi-node training
+- Brings useful tools to help you build a trainer (callbacks, logging, checkpoints)
+- Designed with multi-billion parameter models in mind
```
Comment (justusschock) [inline README.md]: should we mention sharding specifically here? As this is particularly relevant for LLMs

**Review by justusschock — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (awaelchli) [reply README.md]: 🤦

File: README.md (hunk 3 above)
Comment (awaelchli) [reply README.md:4]: I'd like to keep it on latest right now for the time we are incorporating feedback very quickly. Long term I agree, yes we should fix it to a number or stable.

**Pull request merged**